pose_estimate_demo/ImagePoint center
uint16 width
uint16 height
pose_estimate_demo/ImagePoint[4] cornerPoints
